src/sensitivities/ambiguity_resolution.jl

```diff
@@ -0,0 +1,7 @@
+function Base.fill(val::Node, dims::Vararg{Union{Integer,AbstractUnitRange}})
+    return invoke(fill, Tuple{Node, Vararg{Any}}, val, dims...)
+end
+
+function Base.fill(val::Node, dims::Tuple{Vararg{Integer}})
+    return invoke(fill, Tuple{Node, Vararg{Any}}, val, dims)
+end
```

Comment:
Is this funny symbol a style suggestion?  I've not seen it before.

Reply:
I think it is github saying there is no newline as the end of the file.